### PR TITLE
[BZ 1590449] Add support for standalone hawkular-metrics schema installer

### PR DESCRIPTION
**Pull request by rubenvp8510**
This is a backport of https://github.com/openshift/openshift-ansible/pull/7683 to  3.9.

### CODE REVIEW

File: roles/openshift_metrics/tasks/install_hawkular_schema_job.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+- name: list installed jobs
+  command: >
```
Comment (vrutkovs) [inline roles/openshift_metrics/tasks/install_hawkular_schema_job.yaml]: Use `oc_obj` instead

**Comment by vrutkovs** [comment]
bot, test pull request

**Comment by jsanda** [comment]
We need these changes merged into the 3.7 and 3.6 branches as well. Here are the corresponding tickets for reference:

* 3.9 [BZ 1590449](https://bugzilla.redhat.com/show_bug.cgi?id=1590449)
* 3.7 [BZ 1590451](https://bugzilla.redhat.com/show_bug.cgi?id=1590451)
* 3.6 [BZ 1592966](https://bugzilla.redhat.com/show_bug.cgi?id=1592966)

**Comment by vrutkovs** [comment]
>Do not merge yet.

Is this WIP or we're holding until 3.10 release? The master branch is open btw

**Comment by jsanda** [comment]
@vrutkovs The necessary changes for hawkular-metrics have not yet made it into distgit, hence the do not merge msg. As soon as the changes get committed, we will update this PR. Thanks in advance for the review!

**Comment by vrutkovs** [comment]
Looks like GCP is flaking
/test gcp

File: roles/openshift_metrics/templates/hawkular_metrics_schema_job.j2 [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  template:
+    spec:
+      version: v1
+      metadata:
+        labels:
+          metrics-infra: hawkular-metrics
+          #name: hawkular-metrics
+      containers:
+      - name: hawkular-metrics-schema
+        image: {{openshift_metrics_image_prefix}}metrics-schema-installer:{{openshift_metrics_image_version}}
```
Comment (vrutkovs) [inline roles/openshift_metrics/templates/hawkular_metrics_schema_job.j2]: This image doesn't exist, it resolves to `docker.io/openshift/origin-metrics-schema-installer:v3.9` for Origin

Comment (sdodson) [reply roles/openshift_metrics/templates/hawkular_metrics_schema_job.j2]: @rubenvp8510 I think this is blocking the merge of this PR, have you had a chance to check on this?

Comment (rubenvp8510) [reply roles/openshift_metrics/templates/hawkular_metrics_schema_job.j2]: Yes, the scripts for push the metrics-schema image to dockerhub are already updated on master, but the image is not there, still investigating what I need to do or who I need to talk with in order to push the images.

Comment (stevekuznetsov) [reply roles/openshift_metrics/templates/hawkular_metrics_schema_job.j2]: We are publishing images for master, is that not appropriate?

**Pull request closed without merging**